README.md

```diff
@@ -84,19 +84,21 @@ For more information about signing and encrypting messages, see the correspondin
 The `Yiisoft\Mailer\Symfony\Message` class provides a single `getSymfonyEmail()` method that returns
 a [Symfony Email](https://symfony.com/doc/current/mailer.html#creating-sending-messages) instance.
 
-For use in the [Yii framework](https://www.yiiframework.com/), see the configuration files:
-  - [`config/di.php`](https://github.com/yiisoft/mailer-symfony/blob/master/config/di.php)
-  - [`config/params.php`](https://github.com/yiisoft/mailer-symfony/blob/master/config/params.php)
-
 ## Documentation
 
 - [Yii guide to mailing](https://github.com/yiisoft/docs/blob/master/guide/en/tutorial/mailing.md)
 - [Symfony Mailer documentation](https://symfony.com/doc/current/mailer.html)
+- For use in the [Yii framework](https://www.yiiframework.com/), see the configuration files:
```

Comment:
Currently place of this block is better.

Reply:
I didn't understand. What should I do?

Reply:
Block "For use in the yii framework" keep as is.